crates/swc_ecma_minifier/src/compress/optimize/collapse_vars.rs

```diff
@@ -1,73 +1 @@
-use swc_ecma_ast::*;
 
-use super::Optimizer;
```

Comment:
This pass doesn't have any effect. I believe it's completely shadowed by `inline` pass.